src/Microsoft.Health.Dicom.Blob/Features/Storage/BlobFileStore.cs

```diff
@@ -230,16 +241,17 @@ public async Task<Stream> GetFileFrameAsync(long version, string partitionName,
     }
 
     /// <inheritdoc />
-    public async Task<BinaryData> GetFileContentInRangeAsync(long version, Partition partition, FrameRange range, CancellationToken cancellationToken)
+    public async Task<BinaryData> GetFileContentInRangeAsync(long version, Partition partition, FileProperties fileProperties, FrameRange range, CancellationToken cancellationToken)
     {
         EnsureArg.IsNotNull(range, nameof(range));
         EnsureArg.IsNotNull(partition, nameof(partition));
-        BlockBlobClient blob = GetInstanceBlockBlobClient(version, partition.Name);
-        _logger.LogInformation("Trying to read DICOM instance fileContent with watermark '{Version}' on range {Offset}-{Length}.", version, range.Offset, range.Length);
+        BlockBlobClient blob = GetInstanceBlockBlobClient(version, partition, fileProperties);
```

Comment:
Why only for update? so if someone changes the file, frame range will be broken. Don't we need to put the same logic in `GetFileFrameAsync`?